.npmignore

```diff
@@ -0,0 +1,2 @@
+test
+client
```

Comment:
when publishing the package for command line usage, 
- client directory should contain web client specific code, therefore publishing it with the npm package would increase the size of the package.